lib/vssolution.cpp

```diff
@@ -1725,10 +1725,10 @@ void VisualizationSceneSolution::PrepareLines()
 
          MFEM_ASSERT(pointmat.Size() == 4, "Not a quadrilateral!");
 
-         //we assume that mesh_course is used only for tensor finite elements,
-         //like for representation of quadratures, so in 2D it is square
+         // we assume that mesh_course is used only for tensor finite elements,
+         // like for representation of quadratures, so in 2D it is square
          const int geom =
-            Geometry::Type::SQUARE; //ref.embeddings[i].geom; //<---- bugged!?
+            Geometry::Type::SQUARE; // ref.embeddings[i].geom; // <-- bugged!?
```

Comment:
Do we need to leave this here?

Reply:
Yes or I mean without the comment maybe 😄 , but it was bugged (and it is in the referenced MFEM v4.7), see mfem/mfem#4414 .